Makefile

```diff
@@ -147,17 +147,20 @@ install-tools:
 check: install-tools static tidy check-plugin errdoc check-testing-t
 
 static: install-tools
-	@ # Not running vet and fmt through metalinter becauase it ends up looking at vendor
 	@ echo "gofmt ..."
 	@ gofmt -s -l -d $(PACKAGE_DIRECTORIES) 2>&1 | awk '{ print } END { if (NR > 0) { exit 1 } }'
 	@ echo "golangci-lint ..."
 	@ golangci-lint run $(PACKAGE_DIRECTORIES)
 	@ echo "revive ..."
-	@revive -formatter friendly -config revive.toml $(PACKAGES)
+	@ revive -formatter friendly -config revive.toml $(PACKAGES)
+
+	@ for mod in $(SUBMODULES); do cd $$mod && $(MAKE) static && cd - > /dev/null; done
 
 tidy:
-	go mod tidy
-	git diff --quiet go.mod go.sum
+	@ go mod tidy
+	@ git diff --quiet go.mod go.sum
```

Comment:
I suggest adding some output info for this command, otherwise, it will be kind of confusing for a new-bee developer to know where the thing goes wrong.

Reply:
ok